examples/tfengine/generated/multi_envs/cicd/terraform.tfvars

```diff
@@ -65,7 +65,9 @@ envs = [
         run_on_schedule = ""
       }
       plan = {
-        skip = true
+        skip            = true
+        run_on_push     = true
```

Comment:
Actually from looking at this generated config. I think it's less confusing if `run_on_push` is set to `false` when `skip` is set to true. Maybe reverting back to https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/972/commits/00a09cfbf08c09438ea1a8166b02ef38391363ce is a bit better. Sorry for the back and forth.

Reply:
np, I agree.

I did a reset to 00a09cf